examples/inverse/plot_morph_surface_stc.py

```diff
@@ -0,0 +1,72 @@
+"""
+=============================
+Morph surface source estimate
+=============================
+
+This example demonstrates how to morph an individual subject **surface source
+estimate** to a common reference space. For this purpose we will use
```

Comment:
This example demonstrates how to morph an individual subject **surface source
￼estimate** to a common reference space.